sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/LogicalRelation.scala

```diff
@@ -72,9 +72,12 @@ case class LogicalRelation(
   // expId can be different but the relation is still the same.
   override lazy val cleanArgs: Seq[Any] = Seq(relation)
 
-  @transient override lazy val statistics: Statistics = Statistics(
-    sizeInBytes = BigInt(relation.sizeInBytes)
-  )
+  // statistics inherited from a MetastoreRelation
+  @transient var inheritedStats: Option[Statistics] = None
```

Comment:
The `ExternalCatalog` APIs define the methods to interacts with the external catalog. Spark 2.0 provides two implementations, including `HiveExternalCatalog` and `InMemoryCatalog`. The default is `InMemoryCatalog` if the build is not with the Hive profile. Users are able to run `ANALYZE TABLE` against the tables when we using `InMemoryCatalog`. 

You can try this command when you using `InMemoryCatalog`. You might understand what I said above.